### PR TITLE
[BAD-140] - Compile HDP 2.2 shim

### PR DESCRIPTION
**Pull request by andrei-viaryshka**

### CODE REVIEW

**Comment by mbradbury2014** [comment]
@brosander , @mattyb149 Could one of you guys please take a look?

File: ivy.xml [hunk 1]
```diff
@@ -55,6 +55,9 @@
     <dependency conf="shim->default" org="pentaho" name="pentaho-hadoop-shims-hdp21-package" rev="${dependency.hadoop-shims-hdp21.revision}" changing="true">
       <artifact name="pentaho-hadoop-shims-hdp21-package" type="zip"/>
     </dependency>
+    <dependency conf="shim->default" org="pentaho" name="pentaho-hadoop-shims-hdp22-package" rev="${dependency.hadoop-shims-hdp22.revision}" changing="true">
+      <artifact name="pentaho-hadoop-shims-hdp22-package" type="zip"/>
```
Comment (mattyb149) [inline ivy.xml]: See comment from pentaho-hadoop-shims PR 195, should this package replace HDP 2.1?

Comment (mattyb149) [inline ivy.xml]: I talked to Michelle, in the Big Data plugin hdp22 should replace hdp21

**Comment by andrei-viaryshka** [comment]
@mattyb149 Could you please take a look? The file is updated.

**Pull request merged**